fili-core/src/test/groovy/com/yahoo/bard/webservice/web/responseprocessors/CacheV2ResponseProcessorSpec.groovy

```diff
@@ -152,7 +156,38 @@ class CacheV2ResponseProcessorSpec extends Specification {
         then:
         2 * next.getResponseContext() >> responseContext
         1 * next.processResponse(json, groupByQuery, null)
-        0 * dataCache.set(cacheKey, '[]')
+        0 * dataCache.set(*_)
+    }
+
+    def "Overly long data doesn't cache and then continues"() {
+        setup: "Save the old max-length-to-cache so we can restore it later"
+        String max_druid_response_length_to_cache_key = SYSTEM_CONFIG.getPackageVariableName(
```

Comment:
If we decided to comment out the variable, might need to include a default MAX_LONG here.